lib/class-sp-search.php

```diff
@@ -114,8 +114,8 @@ public function get_posts() {
 		} else {
 			$ids = $this->pluck_field( 'post_id' );
 			$this->posts = get_posts( array(
-				'post_type'      => 'any',
-				'post_status'    => 'any',
+				'post_type'      => array_values( get_post_types() ),
```

Comment:
i would flip these two back to 'any', if we're doing a query by $id, there should be nothing else to impede that in some edge case where there is an id we want that for some reason had a post type or status that isn't returned by those functions